### ISSUE TITLE
Postscript for url in howpublished

### ISSUE THREAD

**Issue by tsteffen11** [retorquere/zotero-better-bibtex#951]
Hi,

I try to use the postscript:

```
if (Translator.BetterBibTeX && item.itemType === 'webpage') {
    if (item.accessDate) {
      this.add({ name: 'note', value: "(accessed " + item.accessDate + ")" });
    }
    if (item.url) {
      this.add({ name: 'howpublished', bibtex: "{\\url{" + this.enc_verbatim({value: item.url}) + "}}" });
    }
  }
```

I found this here https://retorque.re/zotero-better-bibtex/scripting/ and added it in the Advanced tab of Better BibTex in the Zotero Preferences. However, I do not seem to get it to work as my Better Bibtex exports are still showing the howpublished field without embedded \url{}. 

Any suggestions? Your help is much appreciated, as this is the first time I try to use a postscript.

Cheers,

T

- [x] I have verified in the Zotero addons screen that I am on the latest release of BBT (https://github.com/retorquere/zotero-better-bibtex/releases/latest) and Zotero
- [x] I am posting a single question, bug or feature request
- [x] I have included a descriptive subject of the problem
- [x] I am available for follow-up questions and testing

# Bug classification

## Exporting references

If your bug report/feature request/question concerns problems with exporting references, please specify whether you're exporting BibTeX or BibLaTeX. Please right-click the references where you experience the export problem, select "Send Better BibTeX Error", and post the resulting report ID (shown in red after you submit) here.

## Non-export problems with BBT

If your issue is a bug report, but not for exports, restart Zotero with debugging enabled (Help -> Debug Output Logging -> Restart with logging enabled), reproduce your problem, and select "Report Better BibTeX error" from the help menu, and post the resulting report ID (shown in red after you submit) here.

**Comment by retorquere** [comment]
Can you

1. From the help menu, restart with debugging enabled
2. Export one reference where you wanted the postscript applied
3. Right-click that reference, select "Report BBT error", and post the resulting debug ID here.

**Comment by tsteffen11** [comment]
Hi retorquere,

Thanks for your reply. Here is the ID R7BVWQ5F.

T

**Comment by blip-bloop** [comment]
:robot: this is your friendly neighborhood build bot announcing [test build 5.0.126.6343.issue-951](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.0.126.6343.issue-951.xpi) ("jurism does not wish to be downloadd...").

**Comment by retorquere** [comment]
Please try again with 6343

**Comment by tsteffen11** [comment]
Here is the new ID: 6MERCJ55. Not sure what you mean by 6343?

Thank you.

T

**Comment by retorquere** [comment]
I meant the test build ("test build 5.0.126.6343.issue-951") linked to above. I think that will solve the issue, if we have confirmation on that, I can cut a new release.

**Comment by tsteffen11** [comment]
That worked. Thank you very much for your help!

T

**Comment by retorquere** [comment]
My pleasure. This will be released in 5.0.128 in a few minutes
